### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by Yiheng0410**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -83,16 +87,16 @@ Format: `help`
 
 Adds a person to the address book.
 
-Format: `add n/NAME i/NRIC p/PHONE_NUMBER e/EMAIL a/ADDRESS r/REMARK d/DATE  [t/TAG]…​`
+Format: `add n/NAME d/DATE i/NRIC p/PHONE_NUMBER e/EMAIL a/ADDRESS r/REMARK d/DATE  [t/TAG]…​`
```
Comment (Impala36) [inline docs/UserGuide.md]: There are two date fields.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -104,7 +108,7 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit INDEX [n/NAME] [i/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/REMARK] [d/DATE] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [d/DATE] [i/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [r/REMARK] [d/DATE] [t/TAG]…​`
```
Comment (Impala36) [inline docs/UserGuide.md]: Same case as above.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -194,7 +198,7 @@ If your changes to the data file makes its format invalid, Police Address Book w
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME i/NRIC p/PHONE_NUMBER e/EMAIL a/ADDRESS r/REMARK d/DATE [t/TAG]…​` <br> e.g., `add n/James Ho i/S1234567A p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 r/shop theft d/12-20-2012  t/friend t/colleague`
+**Add** | `add n/NAME d/DATE i/NRIC p/PHONE_NUMBER e/EMAIL a/ADDRESS r/REMARK d/DATE [t/TAG]…​` <br> e.g., `add n/James Ho i/S1234567A p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 r/shop theft d/12-20-2012  t/friend t/colleague`
```
Comment (Impala36) [inline docs/UserGuide.md]: The example add dat is not in the second position.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -194,7 +198,7 @@ If your changes to the data file makes its format invalid, Police Address Book w
...
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Edit** | `edit INDEX [n/NAME] [i/NRIC] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [r/REMARK] [d/DATE] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
```
Comment (Impala36) [inline docs/UserGuide.md]: Edit and Find date field is not in the second position.

**Review by Impala36 — CHANGES_REQUESTED** [review]
Some changes needs to be addressed.

File: docs/UserGuide.md (hunk 4 above)
Comment (Yiheng0410) [reply docs/UserGuide.md]: done

File: docs/UserGuide.md (hunk 3 above)
Comment (Yiheng0410) [reply docs/UserGuide.md]: done

File: docs/UserGuide.md (hunk 1 above)
Comment (Yiheng0410) [reply docs/UserGuide.md]: removed the duplicated one

File: docs/UserGuide.md (hunk 2 above)
Comment (Yiheng0410) [reply docs/UserGuide.md]: corrected

**Review by Impala36 — APPROVED** [review]
Ok, seems good now.

**Pull request merged**